community/amfora/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+install -Dm755 -t "$1"/usr/bin amfora
```

Comment:
style guide: quote the whole path instead of just `$1`

Reply:
Fixed